manifests/ntpd.pp

```diff
@@ -0,0 +1,19 @@
+# Configure NTP daemon
+class foreman_network::ntpd {
+  if ($foreman_network::configure_ntpd) {
+
+    exec { 'NTP sync':
```

Comment:
it seems that example42/puppet-ntp offers some control over syncing via `runmode'` option. With
```
runmode => 'server',
service_autorestart => true
```
it should force sync. It has some more deps though...
https://github.com/example42/puppet-ntp